### PR TITLE
User Endpoint

### PR DESCRIPTION
**Pull request by kjgilder**
New user endpoint to add users into the Mongo database. (can adjust the schema as necessary moving forward)

### CODE REVIEW

File: api/.env [hunk 1]
```diff
@@ -1 +1 @@
-MONGO_URI=mongodb+srv://User2:kanersdoghouse@bare-bones.pxrwg.mongodb.net/FlatFairDB?retryWrites=true&w=majority&appName=Bare-Bones
+MONGO_URI=mongodb+srv://kjgilder:2htuyRZ09zyS1m0R@pwwusers.8ssyv.mongodb.net/?retryWrites=true&w=majority&appName=PWWUsers
```
Comment (lolitaroz) [inline api/.env:1]: eventually we will wanna switch to a DB they own but for now this is great

File: api/package.json [hunk 2]
```diff
@@ -2,11 +2,11 @@
   "name": "api",
   "version": "1.0.0",
   "description": "",
-  "main": "src/index.ts",
+  "main": "src/index.js",
```
Comment (lolitaroz) [inline api/package.json:5]: wait did we change it to js

**Review by lolitaroz — COMMENTED** [review]
looks great to me! awesome job! one thing i'll say is that idk abt changing the package json to look at javascript rather than typescript, let's wait for @charlotteconze to look at it too and we can merge!!

**Review by charlotteconze — DISMISSED** [review]

**Comment by lolitaroz** [comment]
looks good, we should probably not push the .env file though and instead text each other the API keys

**Review by lolitaroz — APPROVED** [review]
awesome !!!

**Pull request merged**